include/wil/filesystem.h

```diff
@@ -1043,8 +1050,219 @@ namespace wil
         THROW_IF_FAILED(GetFileInfoNoThrow<infoClass>(fileHandle, result));
         return result;
     }
+
+    // Helpers to make the CreateFileW API easier to use. This segregates the OPEN_EXISTING "open an existing file"
+    // cases from the "create a new file" that has 4 variations represented in the create_file_create_behavior enum.

```

Comment:
Is `create_file_create_behavior` a remnant of a previous version of this change that got removed?